src/main/kotlin/solve/importer/view/ChoosingDirectoryView.kt

```diff
@@ -26,31 +28,56 @@ class ChoosingDirectoryView : Fragment() {
         path.bindBidirectional(controller.directoryPath)
         project.bindBidirectional(controller.project)
         top {
-            borderpane {
-                left {
-                    vbox {
-                        label("Project directory")
-                        label {
-                            maxWidth = 300.0
-                            textOverrun = OverrunStyle.ELLIPSIS
-                            path.onChange {
-                                tooltip(path.value)
+            vbox {
```

Comment:
Decompose this View into few smaller views like. Check out how it implemented in `CatalogueView`.